### DISCUSSION TITLE
Google is stopping sideloading on Android 13 or higher when installing lazer(other apps also)

### DISCUSSION THREAD

**Discussion by itsskyballs** [ppy/osu#29947] [Q&A] [answered]
This might be the frustrating day when it comes to destroying sideload
I tried to install osu!lazer and i got an error that says its not compatible or install it on play store
This is because Google wants our privacy safe and there's no way to disable it
See https://www.androidauthority.com/android-15-restricted-settings-sideloading-3481098/ for more details
This destroying sideload thing also affects other Google-free Android custom ROMs

So if you're planning to publish lazer to play store,do it.

**Comment by cihe13375** [comment]
I don't have an android 13+ device but from that webpage it seems that
1) it only applies to apps that request certain permissions, like device manager or reading notifications
2) the protection *can* be disabled even on android 15

Also at least from this discussion https://github.com/ppy/osu/issues/25189 I think it should install on android 14 in general.

**Comment by Joehuu** [comment]
Hmm, out of curiosity, I upgraded to Android 15 beta on my Pixel 6a and don't see anything different from previous versions (14, 13, etc.). It's just the usual "install unknown apps" permission that I already allowed on the FIles app.

> I tried to install osu!lazer and i got an error that says its not compatible or install it on play store

Curious to know what you're seeing on screen.

**Comment by itsskyballs** [comment]
> Hmm, out of curiosity, I upgraded to Android 15 beta on my Pixel 6a and don't see anything different from previous versions (14, 13, etc.). It's just the usual "install unknown apps" permission that I already allowed on the FIles app.
> 
> > I tried to install osu!lazer and i got an error that says its not compatible or install it on play store
> 
> Curious to know what you're seeing on screen.

It really depends on your device whether if its a newer device or a patch update for your phone

**Comment by itsskyballs** [comment, marked as answer]
Maybe i was wrong
osu just installed successfully with no errore
